common/src/main.ext__if_app

```diff
@@ -13,7 +13,7 @@ import { MyApp } from './my-app';
 // They are for sharedStyles in shadowDOM.
 // However, css files imported in other js/ts files are processed by style-loader.
 // @endif
-// import shared from './shared./* @if css */css/* @endif *//* @if less */less/* @endif *//* @if sass */scss/* @endif */';
+// import shared from './shared./* @if css */css/* @endif *//* @if sass */scss/* @endif */';
```

Comment:
👍

Reply:
You reminded me we only have webpack config ready for the shared css for shadow-dom :-(